src/gf.c

```diff
@@ -69,12 +69,18 @@ void jl_call_tracer(tracer_cb callback, jl_value_t *tracee)
 
 /// ----- Definitions for various internal TypeMaps ----- ///
 
-static const struct jl_typemap_info method_defs = {
-    1, &jl_method_type
-};
-static const struct jl_typemap_info lambda_cache = {
-    0, &jl_method_instance_type
-};
+static struct jl_typemap_info method_defs = {0};
+static struct jl_typemap_info lambda_cache = {0};
+
+// The Windows linker is unable to grab the imported `jl_method_type` and `jl_method_instance_type`
+// values at link-time, so we use a constructor method to do the importing ourselves.
```

Comment:
We don't have any actual use for these values, they're present as comments, so they can be turned into text

Reply:
I'm not sure what this means.

Reply:
You can just delete all this